.github/workflows/deploy-v2-review-app.yml

```diff
@@ -0,0 +1,62 @@
+name: Deploy v2 Review App
```

Comment:
Is this workflow required sine we already have deploy-v2 which looks like it does the same?